qiskit/compiler/assemble.py

```diff
@@ -60,6 +60,7 @@ def assemble(experiments: Union[QuantumCircuit, List[QuantumCircuit], Schedule,
              rep_time: Optional[float] = None,
              parameter_binds: Optional[List[Dict[Parameter, float]]] = None,
              parametric_pulses: Optional[List[str]] = None,
+             init_qubits: Optional[bool] = None,
```

Comment:
What should this default be?

Reply:
The default should be `None` by default as you have it. This will leave the door open to defining this behavior in Qiskit by inserting resets at a later date.